questionnaire/questionnaire-service.js

```diff
@@ -23,6 +25,11 @@ function createQuestionnaireService(spec) {
 
     AjvErrors(ajv);
 
+    ajv.addFormat('x-mobilePhoneNumber', mobilePhoneNumber => {
+        const mobilePhoneNumberValidator = createMobilePhoneNumberValidatorService();
+        return mobilePhoneNumberValidator.isValidMobilePhoneNumber(mobilePhoneNumber);
+    });
+
```

Comment:
@sinclairs `ajv` is defined just above this within the same scope. This is the earliest that I can define a new "format" for AJV. Is this what you were originally meaning or is there another issue you were talking about?